### ISSUE TITLE
[AI Modeling] Train models

### ISSUE THREAD

**Issue by roby238** [Intel-Edge-AI-SW-Developers-2nd-Team-1/BoardMaestro#13]
1. Dataset 구조
```
from using following command 
(.otx)$ ds_count ./splitted_dataset 2
./splitted_dataset/: 825
./splitted_dataset/train: 660​
./splitted_dataset/train/pass: 348​
./splitted_dataset/train/fail: 312​
./splitted_dataset/val: 165
./splitted_dataset/val/pass: 87​
./splitted_dataset/val/fail: 78​
```

2. Training 결과
|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Loss|
|----|----|----|----|----|----|----|

3. Details
- how many classes
- evaluation
